tcclient/http.go

```diff
@@ -13,7 +13,7 @@ import (
 	"reflect"
 
 	"github.com/taskcluster/httpbackoff"
-	"github.com/tent/hawk-go"
+	hawk "github.com/tent/hawk-go"
```

Comment:
I don't think I understand this comment so please don't be mad, but the package name I found in that repo is 'hawk'....why can't the repo name have a hyphen? I'm probably misunderstanding something.